acl/types.proto

```diff
@@ -176,7 +176,7 @@ message EACLRecord {
     // Target subject's role class
     Role role = 1 [json_name="role"];
 
-    // List of public keys to identify target subject
+    // List of public keys or accounts (scripthash from public key) to identify target subject
```

Comment:
have we decided if it is 25 or 20 bytes? tbh, i do not understand it from "scripthash from public key". also, i think it is critical to mention lengths now since it must define how to interpret this field

Reply:
`from public key` is definitely wrong in general (multisigs). I'd say
```suggestion
    // List of 20-byte account script hashes to identify target subjects. 33-byte public keys are also supported, however, they are deprecated and script hashes should be derived from them.
```

Reply:
We identify users with 25-byte value in NeoFS (see `OwnerID`). While it's a pure accidental choice and I really don't like it, these additional 5 bytes can be leveraged in various ways (for example, to support Bane accounts), so I propose using the same 25-byte IDs here. This would make it more consistent with the other parts of the protocol and also leave some space for future extensions.